enterprise/server/test/integration/podman/podman_test.go

```diff
@@ -161,6 +161,9 @@ func TestRunHelloWorld(t *testing.T) {
 	}
 	c, err := provider.New(ctx, &container.Init{Props: props})
 	require.NoError(t, err)
+	t.Cleanup(func() {
+		c.Remove(context.Background())
```

Comment:
This should be:

```
err := c.Remove(ctx)
require.NoError(t, err)
```

Here and below

Reply:
Ok I was a little confused here because I'm used to using t.Context() instead of `context.Background()` for tests. And `t.Context()` is cancelled before `t.Cleanup` runs.

It seems like we are using `context.Background()` in most of these tests, which means it's fine to use the same context in the CleanUp func?

As for the `require.NoError`, I didn't think it would have affected the test results inside t.Cleanup, but it seems like it does.

Will change to `require.NoError(t, c.Remove(ctx))`.